Ports/ninja/package.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash ../.port_include.sh
+port=ninja
+version=1.8.2
+workdir=ninja-1.8.2
+files="https://github.com/ninja-build/ninja/archive/v1.8.2.tar.gz ninja-v1.8.2.tar.gz"
+
+build() {
+    CXX=i686-pc-serenity-g++ CXXFLAGS="--sysroot=${SERENITY_ROOT}/Build/Root" \
+    LDFLAGS="--sysroot=${SERENITY_ROOT}/Build/Root" \
+    # platform=linux is close enough.
```

Comment:
Question: why not add proper Serenity support to `configure.py`? As far as I can see `'linux'` is only mentioned in a few places, and the only thing it actually matters for is whether the system supports `ppoll`...

Reply:
Because it's as you say not needed. Fewer patches seems good.